rpcs3/Emu/Cell/SPUASMJITRecompiler.cpp

```diff
@@ -1043,10 +1043,25 @@ void spu_recompiler::IRET(spu_opcode_t op)
 
 void spu_recompiler::BISLED(spu_opcode_t op)
 {
-	fmt::throw_exception("Unimplemented instruction" HERE);
+	//c->mov(auto t, SPU_OFF_32(gpr, op.ra, &v128::_u32, 3));
+	//c->and_(t, 0x3fffc);
+	
+	const XmmLink& vr = XmmAlloc();
+	c->movdqa(vr, XmmConst(_mm_set_epi32(spu_branch_target(m_pos + 4), 0, 0, 0)));
+	c->movdqa(SPU_OFF_128(gpr, op.rt), vr);
+	//todo: external event check
+	//if (external event){
+	//if (op.d || op.e) c->or_(*addr, op.e << 26 | op.d << 27); // interrupt flags stored to PC
+	// c->mov(SPU_OFF_32(pc), *addr);}
+	//}
+	//else
+	//c->mov(SPU_OFF_32(pc),SPU_OFF_32(gpr, vr, &v128::_u32, 3));
+	c->mov(SPU_OFF_32(pc),spu_branch_target(m_pos + 4));
+	//c->unuse(vr);
+	LOG_ERROR(SPU, "BISLED External event");
```

Comment:
LOG_ERROR(SPU, "BISLED External event"); Realy should be called only if external event happen. I supose we don't know what is external event so text by me should be "BISLED External event unimplemented".

Reply:
👍

Reply:
What do you think should be writed here? Can this function be executed when we have internal event?